mqtt/mqtt-bridge/src/upstream/rpc/remote.rs

```diff
@@ -205,164 +209,56 @@ impl RpcHandler {
 }
 
 #[async_trait]
-impl EventHandler for RpcHandler {
+impl EventHandler for RemoteRpcHandler {
     type Error = RpcError;
 
-    async fn handle(&mut self, event: &Event) -> Result<Handled, Self::Error> {
-        match event {
-            Event::Publication(publication) => {
-                if let Some(command_id) = capture_command_id(&publication.topic_name) {
-                    let doc = Document::from_reader(&mut publication.payload.clone().reader())?;
-                    match bson::from_document(doc)? {
-                        VersionedRpcCommand::V1(command) => {
-                            self.handle_command(command_id, command).await?;
-
-                            return Ok(Handled::Fully);
-                        }
-                    }
-                }
-            }
-            Event::SubscriptionUpdates(subscriptions) => {
-                let mut handled = 0;
-                for subscription in subscriptions {
-                    if self.handle_subcription_update(subscription).await? {
-                        handled += 1;
-                    }
-                }
-
-                if handled == subscriptions.len() {
-                    return Ok(Handled::Fully);
-                } else {
-                    return Ok(Handled::Partially);
-                }
-            }
-            _ => {}
-        }
-
-        Ok(Handled::Skipped)
+    async fn handle(&mut self, _event: &Event) -> Result<Handled, Self::Error> {
+        todo!()
+        // match event {
+        //     Event::Publication(publication) => {
+        //         if let Some(command_id) = capture_command_id(&publication.topic_name) {
+        //             let doc = Document::from_reader(&mut publication.payload.clone().reader())?;
+        //             match bson::from_document(doc)? {
+        //                 VersionedRpcCommand::V1(command) => {
+        //                     self.handle_command(command_id, command).await?;
+
+        //                     return Ok(Handled::Fully);
+        //                 }
+        //             }
+        //         }
+        //     }
+        //     Event::SubscriptionUpdates(subscriptions) => {
+        //         let mut handled = 0;
+        //         for subscription in subscriptions {
+        //             if self.handle_subcription_update(subscription).await? {
+        //                 handled += 1;
+        //             }
+        //         }
+
+        //         if handled == subscriptions.len() {
+        //             return Ok(Handled::Fully);
+        //         } else {
+        //             return Ok(Handled::Partially);
+        //         }
+        //     }
+        //     _ => {}
+        // }
+
+        // Ok(Handled::Skipped)
```

Comment:
This was commented because it was moved to LocalRpcHandler? What is the plan here?

Reply:
yeah, this module is to be implemented. part of the logic moved to LocalRpcHandler and the rest will be in RemoteRpcHandler